### PR TITLE
Update for Bio-Formats 6.0.0

### PR DESCRIPTION
**Pull request by dgault**
See https://github.com/ome/bio-formats-documentation/pull/76 for corresponding version history

### CODE REVIEW

File: _posts/2019-02-18-bio-formats-6-0-0.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+Today we are releasing Bio-Formats 6.0.0. This is a major update that includes support for the updated OME-TIFF file format, which now supports multi-resolution tiled images (so-called pyramidal file format). For more info, see http://blog.openmicroscopy.org/file-formats/community/2018/11/29/ometiffpyramid/.  This new version of Bio-Formats also includes support for the KLB format for light sheet microscopy. 
+
+Bio-Formats API changes:
+
+* Java 8 is now the minimum supported version
+* Sub-resolution reading:
+  * added MetadataList and CoreMetadataList classes
+  * added a new SubResolutionFormatReader abstract class for handling pyramidal format readers
+  * updated all pyramid format readers to use SubResolutionFormatReader
+  * deprecated getCoreMetadataList, seriesToCoreInde`, coreIndexToSeries, getCoreIndex and setCoreIndex in IFormatWriter
```
Comment (mtbc) [inline _posts/2019-02-18-bio-formats-6-0-0.md]: `seriesToCoreIndex`

File: _posts/2019-02-18-bio-formats-6-0-0.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+* added a new IPyramidHandler interface with the resolution getter methods
+* Sub-resolution writing changes:
+  * IFormatWriter now extends IPyramidHandler (breaking)
+  * added setResolutions and getResolutions methods to IFormatWriter (breaking)
+  * added examples of using the sub-resolution writing API
+* Tiled writing API changes:
+  * updated IFormatWriter to use setTileSizeX(0) and setTileSizeY(0) as a way to disable tiling (breaking)
+  * updated FormatWriter set 0 as the default values of getTileSizeX() and getTileSizeY (breaking)
```
Comment (mtbc) [inline _posts/2019-02-18-bio-formats-6-0-0.md]: "updated FormatWriter set 0 as ..." feels awkward

File: _posts/2019-02-18-bio-formats-6-0-0.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+* added macro option to always skip LUT writing
+ 
+MATLAB toolbox improvements:
+ 
+* improved performance of bfGetPlane by removing an unnecessary data copy (thanks to Cris Luengo)
+
+Command-line tools improvements:
+
+* bfconvert utility
+  * added `-no-flat` option to the command-line tools to convert files with  sub-resolutions
```
Comment (mtbc) [inline _posts/2019-02-18-bio-formats-6-0-0.md]: double space

**Comment by dgault** [comment]
Thanks @mtbc, good spots

**Pull request merged**